pkg/agent/cniserver/pod_configuration_windows.go

```diff
@@ -38,38 +41,26 @@ func (pc *podConfigurator) connectInterfaceToOVSAsync(ifConfig *interfacestore.I
 		return fmt.Errorf("container lock cannot be null")
 	}
 	ovsPortName := ifConfig.InterfaceName
-	expectedEp, ok := pc.ifConfigurator.getEndpoint(ovsPortName)
+	_, ok := pc.ifConfigurator.getEndpoint(ovsPortName)
 	if !ok {
 		return fmt.Errorf("failed to find HNSEndpoint %s", ovsPortName)
 	}
-	hostIfAlias := fmt.Sprintf("%s (%s)", util.ContainerVNICPrefix, ovsPortName)
+	param := asyncConnectOVSParam{ifConfig, containerAccess}
+	pc.ifConfigurator.addAsyncFuncRequest(ovsPortName, pc.asyncAttachOVSInterface, param)
```

Comment:
Please add comments to explain, why it needs to be async.

Reply:
The existing code is using async solutions to attach the interface to OVS bridge, See https://github.com/antrea-io/antrea/blob/main/pkg/agent/cniserver/pod_configuration_windows.go#L100

A difference is, the existing code creates one goroutine for each new Container interface, I use a single goroutine for all async configuration request.